packages/perseus/src/styles/perseus-renderer.less

```
@@ -314,7 +314,7 @@
         tr:nth-child(odd) {
             // Doesn't work in IE8 :(
             td {
-                background-color: #ededed;
+                background-color: #f7f8fa; // Wonder Blocks offWhite
```

I think this is fine, but I think I'd prefer moving this color declaration to `variables.less`. This makes it reusable, easy to change, and is a little more self-documenting.

```suggestion
                background-color: @offWhite;
```

---

I wish Wonder Blocks exported a .less file with these variables. This is a good alternative, though. Thanks!